spot_rl_experiments/configs/config.yaml

```diff
@@ -101,11 +107,14 @@ PLACE_ACTION_SPACE_LENGTH: 4
 SEMANTIC_PLACE_ACTION_SPACE_LENGTH: 9
 SEMANTIC_PLACE_JOINT_BLACKLIST: [3]
 
+# Semantic Place EE
+SEMANTIC_PLACE_EE_ACTION_SPACE_LENGTH: 10
+
 # Open Close Drawer env
 MAX_LIN_DIST_OPEN_CLOSE_DRAWER: 0.0
 MAX_ANG_DIST_OPEN_CLOSE_DRAWER: 3.73 # degrees
 OPEN_CLOSE_DRAWER_ACTION_SPACE_LENGTH: 8
-OPEM_CLOSE_DRAWER_DISTANCE_BETWEEN_EE_HANDLE: 0.75
+OPEN_CLOSE_DRAWER_DISTANCE_BETWEEN_EE_HANDLE: 0.75
```

Comment:
did we change other instances of `OPEM_CLOSE_DRAWER_DISTANCE_BETWEEN_EE_HANDLE` to use `OPEN_CLOSE_DRAWER_DISTANCE_BETWEEN_EE_HANDLE` now?

Reply:
Yes, there was only one instance where this was used in open_close_drawer_env.py and its changed in the recent commit.